packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -104,6 +105,14 @@ export async function activate(context: vscode.ExtensionContext) {
 function registerCommands(
   extensionContext: vscode.ExtensionContext
 ): vscode.Disposable {
+  // Colorize code coverage
+  const statusBarToggle = new StatusBarToggle();
+  const colorizer = new CodeCoverage(statusBarToggle);
+  const colorizerCmd = vscode.commands.registerCommand(
+    'sfdx.force.apex.toggle.colorizer',
```

Comment:
Tied everything under one command so that it's easy to trigger the feature using keyboard shortcuts.